src/components/common/DAppMessage.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+/**
+ * @desc Component that renders DAppMessages.
+ * @type React.Component
+ */
+
+import React from 'react';
+import { View, TouchableOpacity, Text } from 'react-native';
+import { MediaQueryStyleSheet } from 'react-native-responsive';
+
+import GlobalStyles from '../../global/Styles';
+
+type Props = {
+  /**
+   *
+   * @desc Message to be displayed.
+   */
+  message: string,
+  /**
+   *
+   * @desc Time to be displayed.
+   */
+  time: string,
+};
+
+const styles = MediaQueryStyleSheet.create({
+  ...GlobalStyles,
+});
+
+const DAppMessage = ({
+  time, message,
+}: Props) => (
+  <View style={styles.dAppMessage}>
+    <TouchableOpacity testID='Touchable'>
```

Comment:
I don't think we need the whole view to be Touchable. Or there is any reason for that?

Reply:
Since, normal chats have the whole view as touchable, I thought of this. But I can change it. No issues.

Reply:
Yeah, but this component is not the whole view, it's only a part of Bubble, that also handles touches in some way. So, yes, let's better remove that.

Reply:
Cool, will do that